# ID message for each agent
int64 id
int64 team_id
int64 sub_swarm_id
